src/org/omegat/gui/preferences/view/AppearancePreferencesPanel.form

```diff
@@ -37,5 +37,32 @@
         <AuxValue name="JavaCodeGenerator_VariableModifier" type="java.lang.Integer" value="0"/>
       </AuxValues>
     </Component>
+    <Container class="javax.swing.JPanel" name="jPanel1">
+
+      <Layout class="org.netbeans.modules.form.compat2.layouts.DesignFlowLayout"/>
+      <SubComponents>
+        <Component class="javax.swing.JLabel" name="jLabel1">
+          <Properties>
+            <Property name="text" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+              <ResourceString bundle="org/omegat/Bundle.properties" key="MW_OPTIONMENU_APPEARANCE_THEME_LABEL" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+            </Property>
+          </Properties>
+        </Component>
+        <Component class="javax.swing.JComboBox" name="themeSelectCB">
+          <Properties>
+            <Property name="model" type="javax.swing.ComboBoxModel" editor="org.netbeans.modules.form.RADConnectionPropertyEditor">
+              <Connection code="AppearanceController.getComboBoxModel()" type="code"/>
```

Comment:
Avoiding this kind of hidden code is why I prefer to set the model in `initGui` per the other comment.

This code is completely invisible unless you happen to know where in NetBeans to look, so someone can very easily break the GUI without knowing it.

Reply:
Now explicitly configure it from Controller.